frontend/scss/custom.scss

```diff
@@ -56,3 +56,17 @@ button span.rangle {
     overflow-x: hidden;
   }
 }
+
+/* Make youtube full width */
+.youtube-wrapper {
+  position: relative;
+  width: 100%;
+  height: 0;
+  padding-bottom: 56%;
+}
```

Comment:
@frankharkins you may want to add `margin-bottom` to give the a bit more room between the video and the next paragraph:

<img width="1083" alt="image" src="https://user-images.githubusercontent.com/13156555/134061267-774f2269-2de1-4485-ae54-3f47094120f6.png">

Reply:
Thank you!